Extension/src/LanguageServer/settingsTracker.ts

```diff
@@ -42,18 +42,17 @@ export class SettingsTracker {
 
         for (let key in settingsResourceScope) {
             let curSetting: any = util.packageJson.contributes.configuration.properties["C_Cpp." + key];
-            if (curSetting === undefined) {
-                continue;
-            }
-            let settings: vscode.WorkspaceConfiguration = (curSetting.scope === "resource" || curSetting.scope === "machine-overridable") ? settingsResourceScope : settingsNonScoped;
+            let settings: vscode.WorkspaceConfiguration = (!curSetting || (curSetting.scope === "resource" || curSetting.scope === "machine-overridable")) ? settingsResourceScope : settingsNonScoped;
```

Comment:
Give conditionals at lines 45 and 54 some variable so they're easier to read?

For example:
```
bool isSomethingTrue = (conditional statement here);
let settings: vscode.WorkspaceConfiguration = isSomethingTrue  ? settingsResourceScope : settingsNonScoped;

Reply:
I changed it to a lamba function.